config/crds/openstackproviderconfig_v1alpha1_openstackproviderspec.yaml

```diff
@@ -91,6 +111,10 @@ spec:
           type: array
         sshUserName:
           type: string
+        tags:
```

Comment:
metadata seems missing

Reply:
Yeah, I am not sure what the convention for this is to be honest. These have been auto generated up until this point.

Reply:
It's weird that the auto-generation didn't add metadata here. :thinking:

Reply:
done!